modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/edit_template.jsp

```diff
@@ -105,6 +105,7 @@ boolean showHeader = ParamUtil.getBoolean(request, "showHeader", true);
 			<liferay-ui:error exception="<%= TemplateNameException.class %>" message="please-enter-a-valid-name" />
 			<liferay-ui:error exception="<%= TemplateScriptException.class %>" message="please-enter-a-valid-script" />
 			<liferay-ui:error exception="<%= TemplateSmallImageContentException.class %>" message="the-small-image-file-could-not-be-saved" />
+			<liferay-ui:error exception="<%= TemplateSmallImageInvalidURLException.class %>" message="please-enter-a-small-image-with-a-valid-url" />
```

Comment:
@MrJohn1911 did you have a chance to validate this error message with a designer or someone from the product team?

Reply:
No, I just created it, can you tell me with who can i talk about this?

Reply:
@MrJohn1911 You can talk to Julia Lira and Luiz Jardim

Reply:
Ok, thanks!